install.sh

```diff
@@ -321,4 +327,31 @@ if $nonroot; then
     touch $rprefix/SCYLLA-NONROOT-FILE
     systemctl --user daemon-reload
     echo "Scylla non-root install completed."
+elif ! $packaging; then
+    getent passwd scylla || nousr=1
+    getent group scylla || nogrp=1
```

Comment:
These variables are not initialized, so if they are present in the user's environment they will result in incorrect action.

Reply:
Added initialize code.